server.js

```diff
@@ -19,11 +20,78 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// Start defining your routes here, the thing that you want to get from the database. To get to the route
+// in the browser you write "/" and then the name of the route 
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.json({responsMessage: "Let us see if this message will show and if it is possible to search for book titles and authors"});
```

Comment:
It would be nice if the message could give some hints on which endpoints there is, so it would be easier to try them in the browser (: